tidoop-hadoop-ext/pom.xml

```diff
@@ -30,7 +30,7 @@
     <dependency>
       <groupId>org.apache.httpcomponents</groupId>
       <artifactId>httpclient</artifactId>
-      <version>4.2.1</version>
+      <version>4.3.6</version>
```

Comment:
I'd suggest to mention the change in CNR file.

(Not sure if this CNR file is going to be useful in the future in this repo... but let's do it just in case)